src/objects/deck/demo/alignment.twig

```diff
@@ -1,45 +1,27 @@
+{% if columns and columns > 1 and columnsBreakpoint and columnsBreakpoint != 'none' %}
```

Comment:
This diff looks large, but it just updates this demo from being hard-coded to generating 3 items, and accepting args. There should be no practical difference in the rendered story, but it was bugging me that this was the only story the controls didn't affect.